sklearn/multiclass.py

```diff
@@ -1025,6 +1025,31 @@ def fit(self, X, y):
 
         return self
 
+    def predict_proba(self, X):
+        """Return faux probability estimates.
```

Comment:
I don't know what is "faux" probability.

Reply:
As opposed to the true probability, which would result in a (nigh) perfect calibration plot.

If there's a convention to treat `predict_proba()` scores as probabilities for the docs simplicity's sake, I guess we may just go with 'probability estimates'. Is that better?

Reply:
Indeed, all probabilities given by `predict_proba` in scikit-learn are not "true" probabilities and we don't make the distinction. This is a topic on which we prepare some more details: https://github.com/scikit-learn/blog/pull/152